### PR TITLE
Simple email regex string extension

### PR DESCRIPTION
**Pull request by Benuuu**

### CODE REVIEW

**Comment by loganwright** [comment]
Add tag and bump `Intrepid.podspec` as well, then :shipit: 

P.S.

What's your email for trunk and I'll add you so you can push the update to cocoapods. I have a future vision of setting up jenkins to automatically push merges to cocoapods, but for now we have to do it manually.

**Comment by paulrolfe** [comment]
:rocket:

File: SwiftWisdom/Core/StandardLibrary/String/String+EmailValidation.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+extension String {
+    /**
+     At a minimum emails need an @ and . with at least one character preceeding between and after.  The philosophy here
+     is that email validation should primarily be handled by a confirmation email and that the user generally knows
+     what they are inputting.  We are trying to prevent false-negatives.
+
+     - returns: True if the string contains an @ followed by a . with characters before between and after.
+     */
+    public func ip_isValidEmail() -> Bool {
+        return rangeOfString(".@.+\\.", options: .RegularExpressionSearch) != nil
```
Comment (Benuuu) [inline SwiftWisdom/Core/StandardLibrary/String/String+EmailValidation.swift:20]: `.+@.+\\..+` is what it should be

File: SwiftWisdomTests/StandardLibrary/StringTests.swift [hunk 2]
```diff
@@ -22,4 +22,15 @@ class StringTests: XCTestCase {
...
+    func testValidEmail() {
+        XCTAssertFalse("".ip_isValidEmail())
+        XCTAssertFalse("@".ip_isValidEmail())
+        XCTAssertFalse("@.".ip_isValidEmail())
+        XCTAssertFalse("@.com".ip_isValidEmail())
+        XCTAssertFalse("foo@.com".ip_isValidEmail())
+        XCTAssertFalse("@foo.com".ip_isValidEmail())
+        XCTAssertTrue("f@b.c".ip_isValidEmail())
+        XCTAssertTrue("foo@bar.com".ip_isValidEmail())
+    }
```
Comment (Benuuu) [inline SwiftWisdomTests/StandardLibrary/StringTests.swift:34]: With this test
`XCTAssertFalse("f@b.".ip_isValidEmail())`

**Pull request merged**